test-cases/features/scale-cluster.yaml

```diff
@@ -22,3 +21,6 @@ nemesis_interval: 5
 nemesis_class_name: 'ChaosMonkey'
 
 cluster_health_check: false
+
+workaround_kernel_bug_for_iotune: true
+
```

Comment:
pre-commit isn't happy for the blank line in the end.

-https://jenkins.scylladb.com/job/sct-github-PRs-scan/job/scylla-cluster-tests/job/PR-3631/1/console

Reply:
Somehow it always add this blank line automatically...

Reply:
pre-commit ask last blank line